packages/rest/test/integration/rest.server.integration.ts

```diff
@@ -120,7 +126,11 @@ describe('RestServer (integration)', () => {
   });
 
   it('exposes "GET /openapi.json" endpoint', async () => {
-    const server = await givenAServer({rest: {port: 0}});
+    const server = await givenAServer({
+      rest: {
+        port: 0,
+      },
+    });
```

Comment:
This looks like a change of formatting only, can we revert it please? Unless there is a reason for this change, in which case I'd like to hear it :)

Reply:
I use `prettier` to format the code

Reply:
I believe prettier allow multiple formatting styles these days, depending on how you write the code. I.e. it preserves one-liner `{rest: {port: 0}}` if you started with it, or keeps multi-line version if that's what the developer wrote.